torchci/lib/bot/cliParser.ts

```diff
@@ -56,7 +56,10 @@ const rebase = commands.add_parser("rebase", {
 });
 rebase.add_argument("-s", "--stable", {
   action: "store_true",
-  help: "Rebase to viable/strict",
+  help: "Rebase to viable/strict, takes priority over the --branch flag",
```

Comment:
We can be declarative about the constraints on arguments by using [`add_mutually_exclusive_group`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group`). That way the parser will enforce that the user only passes in one of the two arguments, and we don't have explain the precedence.